frontends/api/src/hooks/topics/index.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery } from "@tanstack/react-query"
+import type {
+  TopicsApiTopicsListRequest,
+  LearningResourceTopic,
+} from "../../generated/v1/api"
+import topics from "./keyFactory"
+
+const useTopicsList = (params?: TopicsApiTopicsListRequest) => {
```

Comment:
How did I miss that. It would be good to refactor the hooks directory to reflect the API, ie. I wouldn't expect to find `topics` inside `learningResources` as they're siblings on the API, 
```
/api/v1/learning_resources
/api/v1/topics
```

Reply:
Yeah, I'm not sure what's best to do here.

Partly, but not entirely, I modeled this after how the backend is set up. topics, departments, learning_resources, courses, etc are all in the `"learning resources app"` in django (`learning_resources/views`). I saw partly because `learning_resources_search` is also a separate django app, but is in the same hook directory.